src/App.js

```diff
@@ -2,12 +2,14 @@ import "./App.css";
 import {Route,Routes} from "react-router-dom";
 import MockAPI from "./MockAPI";
 import Homepage from "./pages/Homepage";
+import VideoPage from "./pages/VideoPage";
 
 
 function App() {
   return (
     <div className="App">
       <Routes>
+        <Route path="/videopage" element={<VideoPage/>}/>
         <Route path="/mock-api" element={<MockAPI/>}/>
         <Route path="/" element={<Homepage/>}/>
       </Routes>
```

Comment:
you can have all routes in a separate file then u can import that file in App.js